hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -0,0 +1,486 @@
+"""
+Script to set up communication, generate dataset and perform hand-eye calibration.
+"""
+from pathlib import Path
+import time
+import datetime
+
+import cv2
+import numpy as np
+import zivid
+import rtde.rtde as rtde
```

Comment:
How would I install this?

Reply:
Explained in KB article, assuming the user have this installed